src/hypernova.js

```diff
@@ -44,7 +44,7 @@ const hypernovaConfig = {
   output: {
     ...shared.output,
     libraryTarget: 'commonjs',
-    path: resolve(shared.output.path, 'server'),
+    path: config.output.hypernovaPath,
```

Comment:
No that `resolve` is no longer being used you should be able to bin the require statement on line one -https://github.com/Futurelearn/webpack-config/blob/43bd95538425f106ed5c864b1c18226ecfc5e7bd/src/hypernova.js#L1

Reply:
Good spot, thanks @sebjacobs